go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/openebs/lvm-localpv
 
-go 1.19
+go 1.22.0
```

Comment:
This is causing github actions to fail. Let's not update this until we uniformly upgrade to golang 1.22 throughout.

Reply:
I will revert that

My go version was the cause of this, That's why I have contributed for Nix Shell :)